### ISSUE TITLE
Pretrained model using all three corpus

### ISSUE THREAD

**Issue by shiinoandra** [buriburisuri/speech-to-text-wavenet#63]
First, I would say thank you for making this awesome repository, 
I've been trying to train the model with all the dataset from 3 corpus but it seems will take really long time (need about 30 hours for single epoch. I'm using GTX1070 in SLI).

is it possible to provide a pretrained model with all the three dataset? as the current pretrained model is trained only with VCTK dataset if I'm not wrong

Thank you very much

**Comment by shiinoandra** [comment]
silly me, i haven't configure the CUDA for the tensorflow in the ubuntu no wonder it runs really slow. after i built again from the source it now detect the CUDA lib and the training is running at decent speed. Training the model using all the corpus now, will report back later

**Comment by calufa** [comment]
@shiinoandra Any luck training the model? Would you mind uploading it to dropbox or google drive?

Cheers!

**Comment by PoojitaT** [comment]
Is it possible for you to share the pretrained model which was obtained for all 3 corpuses?

Thanks

**Comment by greigs** [comment]
I'm 40 epochs in to the training with a 1080ti. Will share the entire training folder output when it's done with 50.

**Comment by greigs** [comment]
entire contents of my /train/ directory (4.5GB zipped) : [https://ufile.io/mivf6](https://ufile.io/mivf6)
output from the newly trained model using the same examples...

```
he hoped there would be stoe for dinner turnips and carrits and brose patitoes and fatin lutton pieces to be latled out an thick peppered flower faten sauce
stuffed into you his belly countled him
after early night fall the yellowe lamps whuld light hup pher and there the squalid quarter of the browfls
ilo burti and e good in your mind
nomebut dan freshn knawly is waiting on you could night husband
```

original example results for comparison...
```
he hoped there would be stoo for dinner turnips and charrats and bruzed patatos and fat mutton pieces to be ladled out in th thick peppered flower fatan sauce
stuffid into you his belly counsiled him
after early night fall the yetl lampse woich light hop here and there on the squalled quarter of the browfles
o berty and he god in your mind
numbrt tan fresh nalli is waiting on nou cold nit husband
```

**Comment by sc1311** [comment]
@greigs @PoojitaT  I got this message while trying to download your pre trained model - "The free hosting period for this file has now expired, only premium users can download it".
Can any of you please upload it on google drive or somewhere else where it can be accessible? 
Thanks

**Comment by ebadali** [comment]
@sc1311 did you get the model ?

**Comment by sc1311** [comment]
@ebadali No

**Comment by hobson** [comment]
@greigs could you renew your uploaded file or put it on a public server somewhere?  ufile.io is pure dark pattern (clickbait, ads, data hostage taking). That isn't the sort of place you'd like to share your hard-won model data. I'd be happy to host your 5GB model, if you could figure out a way to get it to me (dropbox, S3, google drive?).

**Comment by ebadali** [comment]
@hobson @greigs that would be much appreciated

**Comment by AwnishIndus** [comment]
@ebadali @hobson @greigs Can you share the trained model, please. Thanks in advance.

**Comment by hobson** [comment]
@greigs Do you still have the pretrained model?

**Comment by uxigene** [comment]
@AwnishIndus @ebadali @hobson Here is the file uploaded by @greigs. https://drive.google.com/open?id=1RpAg48ZFIsPWm3dC5GU34pRWEXkOH1S- I reuploaded it to google drive.

**Comment by GabrielLin** [comment]
Hi @greigs , @MarinescuEvghenii , What is your CTC loss of using all three corpus. I use all three corpus and I got about 103 CTC loss in Epoch 6. After that, the CTC loss became inf. Thanks.
